src/AdvApi32.Desktop/AdvApi32.Helpers.cs

```diff
@@ -246,8 +246,9 @@ public static unsafe void CreateService(string lpBinaryPathName, string lpServic
 
                     fixed (void* lpInfo = new byte[Marshal.SizeOf(descriptionStruct)])
                     {
-                        Marshal.StructureToPtr(descriptionStruct, new IntPtr(lpInfo), false);
-                        if (!ChangeServiceConfig2(svcHandle, ServiceInfoLevel.SERVICE_CONFIG_DESCRIPTION, lpInfo))
+                        var ptr = new IntPtr(lpInfo);
```

Comment:
I think that is because of that compiling problem,. Wait a little I will send a fix to that.